o2a/scripts/prepare.sh

```diff
@@ -32,15 +35,15 @@ while getopts ":c:r:d:m:" OPT; do
             ;;
      esac
 done
+shift $((OPTIND -1))
```

Comment:
While researching what this does I strumbled upon a mention that it can be unsafe, and that it should rather be:
```
shift "$((OPTIND-1))"
```

Source: https://unix.stackexchange.com/a/214151/78408

Reply:
Fyi. Actually this comment is wrong about  quoting OPTIND. Shellcheck correctly detects all cases where quoting is missing. In this case OPTIND is always integer and $((OPTiND-1)) result is always integer so no globbing/word splitting can occur in the first place. But quoting makes no difference in this case so it can be added :)

Reply:
Interesting. Maybe it would be good to comment on the issue in StackExchange to explain this? ;)

Reply:
My thought exactly. Wanted to do it first think I came to the office - (I commented while having breakfast at my phone ;)

Reply:
Actually - I have enough points on SO to be able to EDIT answers. Which I just did.

Reply:
Haha nice! 😎